Source/WebCore/rendering/style/StyleBoxData.h

```diff
@@ -91,11 +91,11 @@ class StyleBoxData : public RefCounted<StyleBoxData> {
 
     int m_specifiedZIndex;
     int m_usedZIndex;
-    unsigned m_hasAutoSpecifiedZIndex : 1;
-    unsigned m_hasAutoUsedZIndex : 1;
-    unsigned m_boxSizing : 1; // BoxSizing
-    unsigned m_boxDecorationBreak : 1; // BoxDecorationBreak
-    unsigned m_verticalAlign : 4; // VerticalAlign
+    PREFERRED_TYPE(bool) unsigned m_hasAutoSpecifiedZIndex : 1;
+    PREFERRED_TYPE(bool) unsigned m_hasAutoUsedZIndex : 1;
+    PREFERRED_TYPE(BoxSizing) unsigned m_boxSizing : 1;
+    PREFERRED_TYPE(BoxDecorationBreak) unsigned m_boxDecorationBreak : 1;
+    PREFERRED_TYPE(VerticalAlign) unsigned m_verticalAlign : 4;
```

Comment:
Almost wish the `unsigned` was part of the macro too, just for brevity. Maybe instead of a `PREFERRED_TYPE` macro there could be a `TYPED_BITFIELD` macro, but I guess that name over promises a bit.

Reply:
One day, c++ reflection will allow us to build the type casting getter/setters too, but not this day.